corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -1008,6 +1029,12 @@ def column_fieldset(self):
     @memoized
     def initial_columns(self):
         if self.existing_report:
+            reverse_agg_map = {
+                'avg': 'Average',
+                'sum': 'Sum',
+                'count': 'Count',
+                'simple': 'No Aggregation'
+            }
             cols = []
```

Comment:
@amsagoff @krsdimagi @calellowitz 
Will let Amelia/Kevin comment on the name for the expanded choice (though my vote is Count Per Choice).  

Amelia and I discuss the "count" option and we wanted to drop it since its behaviour was unclear and we'd like to support it in a different way in the future.